libs/blocks/adobetv/adobetv.js

```diff
@@ -29,13 +37,5 @@ const loadAdobeTv = (a) => {
 
 export default function init(a) {
   a.classList.add('hide-video');
-  if (a.textContent.includes('no-lazy')) {
```

Comment:
Is there some documentation in need of updating once there's no longer a need for `no-lazy`?

Reply:
There never was any docs about it. @seanchoi-dev do you know if this was exclusively used for unit tests? That's what it looked like to me